### ISSUE TITLE
Error from Markdown Preview Enhanced when opening VSCode

### ISSUE THREAD

**Issue by fmartelg** [shd101wyy/markdown-preview-enhanced#1411]
Whenever I launch VSCode I get the following error:

`Error: EPERM: operation not permitted, open 'C:\Users\UserName\.mume\config.json'  
Source: Markdown Preview Enhanced (Extension)`

This only started recently.  It is strange bc the file exists in that location.  Here are it's contents:

`{  
"vscode_mpe_version": "0.3.5"  
}`

I am logged in as UserName (not admin) in UserName account. 

VsCode Version: 1.45.1 (system setup)
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-14T08:27:35.169Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Comment by fmartelg** [comment]
Yes. I deleted it. Reinstalled the extension. The problem is gone.

I guess this solves the symptoms but not the root cause of the problem...

**Comment by sqrl-ca** [comment]
I've been having this issue for a while on my laptop (twoish years old), never my newer desktop.
I also just found out I had an old VS Code install in `C:\Program Files\...` and uninstalled it.
The issue was there both before and after the uninstall, I just never bothered to track it down.

Thank you for posting! 
And of course thank you to the team for the wonderful extension that makes my life communicating with clients easier by being able to quickly convert thorough technical notes into easy-to-copy, customized through CSS, preview, that I can then paste into Outlook, and is the friendliest paste into Outlook I have experienced.
